ethereum/contracts/relayer/coreRelayer/CoreRelayerDelivery.sol

```diff
@@ -326,56 +345,75 @@ abstract contract CoreRelayerDelivery is CoreRelayerGovernance {
     function payRefundRemote(
         IWormholeRelayerInternalStructs.DeliveryInstruction memory instruction,
         uint256 refundAmount,
-        IRelayProvider provider
-    ) internal returns(bool, RefundStatus){
-        IWormhole wormhole = wormhole();
-        uint256 wormholeMessageFee = wormhole.messageFee();
-        uint256 overhead = wormholeMessageFee + provider.quoteDeliveryOverhead(instruction.refundChain);
+        address providerAddress
+    ) internal returns (bool, RefundStatus) {
+        bool isChainSupported;
+        uint256 wormholeMessageFee = wormhole().messageFee();
+        uint256 receiverValueTarget;
+        uint256 maximumBudget;
+        address rewardAddress;
+
+        if (refundAmount <= wormholeMessageFee) {
+            return (false, RefundStatus.CROSS_CHAIN_REFUND_FAIL_NOT_ENOUGH);
+        }
 
-        if (refundAmount > overhead) {
```

Comment:
Where are we ensuring the refund is greater than the overhead in the new version?

Reply:
if refundAmount < overhead, then receiverValueTarget will be 0, which is checked